app/services/booking_bug_connection.rb

```diff
@@ -7,23 +7,21 @@ def initialize(config:)
   end
 
   def auth_token
+    Rails.logger.info('BookingBugConnection') { 'Retrieving authorisation token' }
```

Comment:
I've just had a look at this and it is very verbose, especially in debug mode.

And given this is now the default log level for rails apps not 100% sure this is what we want - at least not in its default form, maybe extract a custom version based on what theirs is the best solution?

thought